addons/mail_group/tests/test_mail_group.py

```diff
@@ -110,6 +110,17 @@ def test_find_member_for_alias(self):
         self.test_group.alias_id.alias_contact = 'followers'
         self.assertFalse(self.test_group._alias_get_error_message({}, msg_dict, self.test_group.alias_id))
 
+    def test_find_group_user_for_alias(self):
+        """Check for mail incoming from an allowed group. Specifically for a situation where the sender is a part of the allowed user group, but is NOT a member of the mailing list."""
```

Comment:
Let's try and keep these lines within a reasonable length ^^

Also test should be ordered alphabetically.

Reply:
spitted long lines, reordered test